example/devServer.js

```diff
@@ -0,0 +1,65 @@
+const path = require('path');
+const exec = require('child_process').exec;
+const Express = require('express');
+const watch = require('node-watch');
+
+const srcPath = __dirname.split('/example')[0] + '/src';
+const examplePath = __dirname.split('/example')[0] + '/example';
+
+// const hotBuild = () => exec('npm run transpile:main', (err, stdout, stderr) => {
+// 	if (err) throw err;
+// 	if (stdout) {
+// 		console.log(`npm run build:dist --- ${stdout}`);
+// 	}
+// 	if (stderr) {
+// 		console.log(`npm run build:dist --- ${stderr}`);
+// 	}
+// });
+
+const hotBuildExample = () => exec('npm run transpile:example', (err, stdout, stderr) => {
+	if (err) throw err;
+	if (stdout) {
+		console.log(`npm run build:dist --- ${stdout}`);
+	}
+	if (stderr) {
+		console.log(`npm run build:dist --- ${stderr}`);
+	}
+});
+
+watch([srcPath, examplePath], {
```

Comment:
Would it make more sense to use `webpack-dev-server` for this?

Reply:
sure, I think use `webpack-dev-server` is simple! I will fix it.